packages/cli/src/lib/utils.ts

```diff
@@ -71,34 +71,22 @@ export const parseBrowser = (browser?: string): string | Error => {
 };
 
 export const getAxeSource = (axePath?: string): string | void => {
-  // Abort if axePath should exist, and it isn't
+  /* User has specified a path to axe-core, check if it exists */
   if (axePath && !fs.existsSync(axePath)) {
     return;
   }
 
-  // Look for axe in current working directory
-  if (!axePath) {
-    axePath = path.join(process.cwd(), 'axe.js');
-  }
+  /* Attempt to look for the axe-core source in the CWD */
+  axePath = path.join(process.cwd(), 'axe.js');
 
   if (!fs.existsSync(axePath)) {
-    // Look for axe in CWD ./node_modules
+    /* Attempt to look for the axe-core source in the CWD node_modules */
     axePath = path.join(process.cwd(), 'node_modules', 'axe-core', 'axe.js');
   }
 
   if (!fs.existsSync(axePath)) {
-    // `__dirname` is /@axe-core/cli/dist/src/lib when installed globally
-    // to access the locally installed axe-core package we need to go up 3 levels
-    // if all else fails, use the locally installed axe
-    axePath = path.join(
-      __dirname,
-      '..',
-      '..',
-      '..',
-      'node_modules',
-      'axe-core',
-      'axe.js'
-    );
+    /* Obtain the axe-core source from the globally installed CLI package instead */
+    axePath = path.join(__dirname, '..', 'node_modules', 'axe-core', 'axe.js');
```

Comment:
Please explain why `require.resolve()` won't work.

Reply:
After looking into it again - we can rely on `require.resolve(..)` we execute a specific file from `dist`: `"axe": "./dist/cli.js"` - We don't _need_ to have ESM here - reverting back to use `require.resolve`